ui/src/app/job-list/table/table.component.ts

```diff
@@ -140,17 +140,33 @@ export class JobsTableComponent implements OnInit {
   onAbortJobs(): void {
     const aborts: Promise<void>[] = [];
     const selected = this.selection.selected.slice();
+
+    let numErrs = 0;
+    const ref = this.errorBar.open(
+      `Aborting jobs...`, /* action */ '',
```

Comment:
nit: Any reason why tickmarks vs. single quotes?

Reply:
Thanks; just copy-paste (back-ticks needed for interpolation elsewhere). Fixed.